### ISSUE TITLE
bootstrap.sh - psql: could not connect to server: No such file or directory

### ISSUE THREAD

**Issue by fmsouza** [EverythingMe/redash#540]
I'm trying to install the tool inside a Docker container. My Dockerfile to build the image just downloads the ```bootstrap.sh``` from the ```master``` branch to my ```/tmp```, gives it run permission and then run the script to build the environment.

However, it runs well until the time to connect in postgres to create user and database (Line 125). Here it breaks. Those are the error messages shown during the script execution:

```
psql: could not connect to server: No such file or directory
	Is the server running locally and accepting
	connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?
Creating redash postgres user & database.
createuser: could not connect to database postgres: could not connect to server: No such file or directory
	Is the server running locally and accepting
	connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?
```

And the Unix domain socket displayed above seems not to exist.
```
root@09b25ed5fdc4:/tmp# cat /var/run/postgresql/.s.PGSQL.5432
cat: /var/run/postgresql/.s.PGSQL.5432: No such file or directory
```

**Comment by arikfr** [comment]
Can you provide the full output of the Docker build run? Also, what Docker base image are you using?

In general, I don't recommend building a docker container with the bootstrap script, as it goes against Docker best practices and runs multiple processes (PostgreSQL, Redis, re:dash web server, re:dash workers) in one container. I'll be happy to collaborate on creating a more idiomatic Docker image.

**Comment by fmsouza** [comment]
My base is ```ubuntu:14.04``` and I'll retry the installation in another container later to provide you the log (I managed to make it work running the script after the container startup, then restarted postgresql service and runned the script again).

I was creating a Dockerfile for personal purpose, as I managed to run it on Google Cloud but would install it on a private server. And yes, I think it would be great to collaborate with you to create a Docker-friendly infrastructure. I'll notice you if I make progress about it.
